components/light/esp32_rmt_led_strip.rst

```diff
@@ -48,6 +38,16 @@ Configuration variables
     - ``BGR``
     - ``BRG``
 
+- **rmt_channel** (**Optional**, int): The RMT channel to use. If using multiple strips, you need to use different channels. Obsolete with ESP-IDF v5.
```

Comment:
The default IDF version is now 5.1.5; instead of calling out "Obsolete with ESP-IDF v5" or "Only with ESP-IDF v5", I think the docs should be written with the assumption that IDF 5.1.5 is what's being used. Put another way: if this isn't necessary configuration anymore, it should be simply be removed.

That said, if there is a legacy configuration variable that's still available/required for IDF < 5.x, _that_ should be called out, but let's maybe move those down to a separate, appropriately-named section below the "main" configuration variables here.

Reply:
Another thought -- since our default Arduino version does not use IDF 5 (yet), it may instead be better to differentiate the configuration variables in terms of what's used/needed for IDF vs. Arduino.

Reply:
How about:

ESP32 IDF configuration variables:
- **clock_resolution** (*Optional*, int): The clock resolution used by the RMT peripheral in hz. Defaults to ``1000000``.

ESP32 Arduino configuration variables:
- **memory_blocks** (*Optional*, int): The number of RMT memory blocks used. The maximum
  number of blocks shared by all receivers and transmitters depends on the ESP32 variant. Defaults to ``3``.

Many of them are labelled esp32 only. Might as well do both